package.json

```diff
@@ -35,6 +35,9 @@
     "the four fifths",
     "msoe"
   ],
+  "dependencies": {
+    "moment-timezone": "^0.5.9"
```

Comment:
NACK: moment-timezone is already in devDependencies; why is this outdated version here too?